### ISSUE TITLE
Aproval inconsitencies

### ISSUE THREAD

**Issue by rryter** [jenkins-x/lighthouse#1253]
The user `rryter` was added to the OWNERS file in this PR. So it makes sense, that he's not yet able to approve via `/approve`. How ever the message still contains the information that the PR was approved by said user.

<img width="956" alt="Screenshot 2021-03-05 at 08 29 51" src="https://user-images.githubusercontent.com/798709/110083153-a3b04b80-7d8e-11eb-852e-d953469eef50.png">

**Comment by noorul** [comment]
I think it is because the author of the PR is rryter

**Comment by hervelemeur** [comment]
If you just added "rryter" to the OWNERS in this PR, it's by design he/you can't approve this PR since it's only in this PR branch code.

It will be possible as soon as "rryter" is in the main branch OWNERS.

If it wasn't the case, anyone could create a PR with their username in OWNERS, and be added to the trusted users without any verification/safeguard.

**Comment by rryter** [comment]
I totally agree @hervelemeur. The issue here is that the text should reflect that. As of right now it says:

This PR is NOT APPROVED.
**This pull-request has been approved by: rryter**

**Comment by noorul** [comment]
@rryter I also had the same confusion.
